src/components/dls/Skeleton/Skeleton.module.scss

```diff
@@ -1,15 +1,7 @@
+@use "src/styles/theme";
 $baseSize: var(--spacing-mega);
 $animationDuration: 8s;
 
-@keyframes loading {
```

Comment:
sorry, I am I didn't follow this PR since the beginning.

but is there a reason we're removing this loading `background-position` animation ?

Reply:
The animation wasn't showing up in dark mode, I could've fixed that but I found Material UI use this approach and it looked better, I commented above with videos to show the difference. wdyt?